featurebyte/service/base_document.py

```diff
@@ -568,6 +581,7 @@ async def list_documents_iterator(
         AsyncIterator[Document]
             List query output
         """
+        assert page_size > 0, "page_size must be greater than 0"
```

Comment:
Curious what is the reason to enforce this? Doing a quick search there are places where `page_size=0` are used.

Reply:
During the development, i unpurposely set this to 0 and it falls into infinite loops. Takes me a while to find this problem. Just to add an assertion here for troubleshooting.